src/translate.ts

```diff
@@ -152,6 +153,7 @@ export async function translatePo(
   verbose: boolean,
   output: string,
   contextFile: string,
+  timeout: number = 20000,
```

Comment:
This line of default value is redundant
```suggestion
  timeout: number,
```